backend/controllers/backLink.ts

```diff
@@ -0,0 +1,37 @@
+import config from '../config'
+
+const sanitizeUrl = (url: string) => (url?.endsWith('/') ? url.substring(0, url.length - 1) : url)
+
+export interface RegisteredService {
+  name: string
+  hostname: string
+  defaultBackLinkText: string
+}
+
+export const registeredBackLinkServices = [
+  {
+    name: 'digital-prison-services',
+    hostname: '',
+    defaultBackLinkText: 'View most recent search',
+  },
+  {
+    name: 'welcome-people-into-prison',
+    hostname: sanitizeUrl(config.apis.welcomePeopleIntoPrison.url),
+    defaultBackLinkText: 'Back to Welcome people into prison',
+  },
+] as Array<RegisteredService>
```

Comment:
you don't need to do a type assertion here with the `as`, if you want to be explicit about the type you can: `export const registeredBackLinkServices:  RegisteredService[] = ...`